File: crates/mct-lang-csharp/tests/fixtures/billing-app/Invoice.cs
namespace Billing {
    public class Invoice {
        private double total;

        public double AddItem(double price) {
            total += price;
            Logger.Log("item added");
            return total;
        }

        public double AddItem(double price, double taxRate) {
            double taxed = price * (1 + taxRate);
            total += taxed;
            Logger.Log("item added with tax");
            return total;
        }
    }
}

File: crates/mct-lang-go/tests/fixtures/billing-app/invoice.go
package billing

type Invoice struct {
	Total float64
}

func (inv *Invoice) AddItem(price float64) float64 {
	inv.Total += price
	Log("item added")
	return inv.Total
}

func (inv *Invoice) AddItemWithTax(price float64, taxRate float64) float64 {
	taxed := price * (1 + taxRate)
	inv.Total += taxed
	Log("item added with tax")
	return inv.Total
}
